packages/docs/getting-started/running-attestation-service.md

```diff
@@ -0,0 +1,125 @@
+# Running the Attestation Service
+
+- [Running the Attestation Service](#running-the-attestation-service)
+  - [Environment variables](#environment-variables)
+  - [Sms Providers](#sms-providers)
+    - [Nexmo](#nexmo)
+    - [Twilio](#twilio)
+  - [Accounts Configuration](#accounts-configuration) \* [Database Configuration](#database-configuration)
+  - [Executing the Attestation Service](#executing-the-attestation-service)
+
+As part of the [lightweight identity protocol](/celo-codebase/protocol/identity), validators are expected to run an Attestation Service to provide attestations that allow users to map their phone number to an account on Celo. The Attestation Service is a simple Node.js application that can be run with a Docker image.
+
+## Environment variables
+
+The service needs the following environment variables:
+
+| Variable        | Explanation                                                                                                                                                            |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| DATABASE_URL    | The URL under which your database is accessible, currently supported are `postgres://`, `mysql://` and `sqlite://`                                                     |  |
+| CELO_PROVIDER   | The URL under which a celo blockchain node is reachable, i.e. something like `https://integration-forno.celo-testnet.org`                                              |  |
+| ACCOUNT_ADDRESS | The address of the validator account                                                                                                                                   |  |
+| ATTESTATION_KEY | The private key with which attestations should be signed. You could use your account key for attestations, but really you should authorize a dedicated attestation key |  |
```

Comment:
We should probably just tell people to delegate an attestation signer, it's actually quite hard to retrieve the account key for use anyways